rfcs/0009_Unified_Static_Memory_Planning.md

```diff
@@ -0,0 +1,467 @@
+    Feature Name: Unified Static Memory Planner
+    Start Date: 2021 June 1
+    RFC PR: #0009
+    GitHub Issue: https://github.com/apache/tvm/issues/8404
+
+# Background
+
+Currently, given a ML model primarily TVM will generate two main artifacts :
+
+* A1 : Description of the sequential execution of operators :
+  1. If the "executor" is "graph", this would be a JSON
+  2. if the "executor" is "aot", this would be a main function describing call graph of operators
+* A2 : library of operators (in the form of runtime.Module)
+
+A1 is generally created out of lowering the "main" relay function and A2 is created lowering fused relay primitive functions → TIR PrimFuncs → C or LLVM artifacts of the operator library.
```

Comment:
Is A1 created from lowering the main function or from lowering the entire relay program?

Reply:
The description of the calls to the operator PrimFuncs could be defined as lowering the "main" relay function (that has calls to the primitive relay functions -- post fuse-ops). I'd view that differently to lowering of the entire relay program that uses the compile engine lower each relay primitive function. Here A1 just refers to the former. Does that make sense?